script/plume-front.sh

```diff
@@ -1,8 +1,24 @@
 #!/bin/bash
 
-ARCH=`arch`
+ARCH=$(python <<EOF
+from __future__ import print_function
+import platform
+processor = platform.machine()
+architecture = platform.architecture()
+if processor == 'aarch64':
+    # Mutli arch arm support is why this 32bit check is present
+    if '32bit' in architecture:
+        print('armv71', end='')
+    else:
+        print('aarch64', end='')
+elif processor == 'x86 64' or processor == 'x86_64':
+    print('amd64', end='')
+elif processor == 'armv7l':
+    print('armhf', end='')
```

Comment:
how big is the chance that anyone's still using i386?

Reply:
@igalic the main linux kernel dev's have suggested dropping i386 support...

If it's something others run into I'm happy to add it back but i need the platform.machine() output to add it to the conditional. I don't have access to i386 and didn't want to add an arch that I don't have valid output for.